Makefile

```diff
@@ -31,6 +31,9 @@ COMMIT_NO_SHORT := $(shell git rev-parse --short HEAD 2> /dev/null || true)
 COMMIT := $(if $(shell git status --porcelain --untracked-files=no),${COMMIT_NO}-dirty,${COMMIT_NO})
 VERSION_COMMIT := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
 ARCH := $(shell go env GOARCH)
+ifeq ($(SECCOMP),yes)
+BUILDTAGS := seccomp
+endif
```

Comment:
The same way we have `INIT := no` by default, please make sure we have `SECCOMP := no` too.
This way, we can merge this PR, but leaving the default way of building the agent the way it is.

Reply:
@jodh-intel @sboeuf : Sorry, I didn't understand the comment completely.  
1. When SECCOMP=yes will be passed when making a local rootfs image, If we use `SECCOMP := no` in this Makefile, it shall override its value, thus having no effect of environment variable passed. 
2. We can set SECCOMP to no only if it's not set, but then still agent would be built the same way `go build -tags "" -o kata-agent -ldflags "-X main.version=1.2.0-5765593309eab087598c093ff226500af158df9c-dirty"`
```
ifeq ($(SECCOMP),)
SECCOMP := no
endif
``` 
Probably I am missing something.

Reply:
Actually, I'm a bit confused now too as your code will work as we want currently.

I think @sboeuf might simply be suggesting that for consistency with the `INIT` option that you supplement your change with an explicit:

```
# Set to 'yes' to build agent with seccomp support
SECCOMP := no
```

Reply:
@nitkon: variables passed from command line have priority over definitions inside Makefiles: https://www.gnu.org/software/make/manual/make.html#Override-Directive

Reply:
@marcov : Oh I was thinking the other way round. Now it makes sense. Update my PR. :-D

Reply:
The only time this *won't* work is if the user wants to enable seccomp and tries to flag that by setting an actual environment variable:

```
$ export SECCOMP="yes"
$ make  # SECCOMP will still be "no" here.
```

Reply:
@jodh-intel: Yea, I had tried it the same way as you mentioned and so thought it to be the other way round. :-)